textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriterTest.java

```diff
@@ -1,6 +1,10 @@
 package edu.uci.ics.textdb.dataflow.queryrewriter;
 
+import edu.uci.ics.textdb.api.common.IField;
```

Comment:
@rajesh9625 Got it! Just realized that `containsAllResults()` takes in ArrayLists and not Lists. Hence my confusion, my apologies. The test now uses `containsAllResults()` as suggested. @chenlica